src/user.js

```diff
@@ -1,38 +1,30 @@
-const Pantry = require('../src/Pantry');
-const Recipe = require('../src/Recipe');
+const Pantry = require('../src/pantry');
+// const Recipe = require('../src/recipe');
 
 class User {
   constructor(obj) {
     this.name = obj.name;
     this.id = obj.id;
-    this.pantry = new Pantry(obj.pantry)
+    this.pantry = new Pantry(obj.pantry);
     this.favorites = [];
-    this.mealToCook = []
   }
 
-  addToFavorites() {
+  addToFavorites(recipe) {
     //adds recipe to the favorites array
+    this.favorites.push(recipe)
 
   }
 
   removeFromFavorites() {
```

Comment:
Beings as we could just keep the add to favorite method to basically a toggle this may not even be needed.